Public/Src/Cache/ContentStore/Library/Service/Grpc/GrpcCopyClient.cs

```diff
@@ -29,6 +28,11 @@ public sealed class GrpcCopyClient : IShutdown<BoolResult>
         private readonly string _host;
         private readonly int _grpcPort;
 
+        /// <summary>
+        /// Gets or sets a value indicating whether compression may be used during transmission.

```

Comment:
It seems that the comment is out of sync. It just returns the indication but can't be changed.

Reply:
Fixed